scripts/mockgen.sh

```diff
@@ -1,8 +1,19 @@
 #!/usr/bin/env bash
 
-# Use this script to regererate generated mock files
+# Use this script to regenerate generated mock files
 # after changing signatures of interfaces in these packages
 
+# check if mockgen is installed
+mockgen --version
+if [ $? -eq 0 ]; then
```

Comment:
You could also test the version installed, as the output differs depending on the version